### PR TITLE
Add layer info button

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/components/LayerInfosDialog/__snapshots__/LayerInfosDialog.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`LayerInfosDialog should match snapshot when Layer is defined 1`] = `null`;
```
Comment (edellucien) [inline src/components/LayerInfosDialog/__snapshots__/LayerInfosDialog.test.js.snap]: snapshot always empty currently. Better use a real layer for testing?

Comment (oterral) [reply src/components/LayerInfosDialog/__snapshots__/LayerInfosDialog.test.js.snap]: that's the point. I want it returns null if there is no layer.

Comment (oterral) [reply src/components/LayerInfosDialog/__snapshots__/LayerInfosDialog.test.js.snap]: Tests fixed

Comment (edellucien) [reply src/components/LayerInfosDialog/__snapshots__/LayerInfosDialog.test.js.snap]: Thanks :)

File: src/components/Dialog/Dialog.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  footer: PropTypes.element,
+};
+
+const defaultProps = {
+  title: null,
+  body: null,
+  footer: null,
+};
+
+const hideDialog = dispatch => () => {
```
Comment (mhaertwig) [inline src/components/Dialog/Dialog.js]: @oterral do we really need this function to call a function with a function? :wink:

Comment (oterral) [reply src/components/Dialog/Dialog.js]: ahah not anymore indeed. Thx for pointing this out

**Pull request merged**